assets/component-cart-notification.css

```diff
@@ -25,10 +21,10 @@
   width: 100%;
   z-index: -1;
   filter: drop-shadow(
-    var(--shadow-horizontal-offset)
-    var(--shadow-vertical-offset)
-    var(--shadow-blur-radius)
-    rgba(var(--color-foreground), var(--shadow-opacity))
+    var(--popup-drawer-shadow-horizontal-offset)
+    var(--popup-drawer-shadow-vertical-offset)
+    var(--popup-drawer-shadow-blur-radius)
+    rgba(var(--color-foreground), var(--popup-drawer-shadow-opacity))
   );
```

Comment:
I see it done as well for the .button, .shopify-challenge__button, .custom button but I guess that'd be a different PR not based on what you did for popup and drawers.